.github/workflows/deploy.yml

```diff
@@ -72,6 +81,30 @@ jobs:
           azure_credentials: ${{ secrets.AZURE_CREDENTIALS }}
           terraform_vars: workspace_variables/dev.tfvars.json
 
+  deploy_review:
+    name: Deploy to review environment
+    concurrency: deploy_review_${{ github.event.pull_request.number }}
+    needs: [docker]
+    runs-on: ubuntu-latest
+    if: contains(github.event.pull_request.labels.*.name, 'deploy') || github.event_name != 'pull_request'
+    environment:
+      name: review
+      url: ${{ steps.deploy.outputs.environment_url }}
+
+    outputs:
+      environment_url: ${{ steps.deploy.outputs.environment_url }}
+
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
This could be moved inside of the deploy action
And removed from https://github.com/DFE-Digital/apply-for-qualified-teacher-status/blob/main/.github/workflows/deploy.yml#L89 as well